gslib/tests/testcase/integration_testcase.py

```diff
@@ -276,6 +276,26 @@ def _SetObjectCustomMetadataAttribute(self, provider, bucket_name,
       self.xml_api.PatchObjectMetadata(bucket_name, object_name, obj_metadata,
                                        provider=provider)
 
+  def get_random_ascii_chars(self, size, seed=None):
```

Comment:
This method can be removed. It was merged in PR #623 , but moved to utils/text_util.py

Reply:
removed